src/compose/volumes/initializer/registerEndpoints.json

```diff
@@ -1,26 +1,56 @@
 [
-  {
-    "aasId": {
-        "id": "SAP_CentralAssetRepository",
-        "idType": "IRI"
+    {
+        "aasId": {
+            "id": "https://i40-test-aas-server.cfapps.eu10.hana.ondemand.com/aas",
+            "idType": "IRI"
+        },
+        "endpoints": [
+            {
+                "url": "https://admin:admin123@i40-test-aas-server.cfapps.eu10.hana.ondemand.com/ingress",
+                "target": "cloud",
+                "protocol": "https",
+                "protocolVersion": "1.1"
+            }
+        ],
+        "assetId": {
+            "id": "https://i40-test-aas-server.cfapps.eu10.hana.ondemand.com",
+            "idType": "IRI"
+        }
+    },
+    {
+        "aasId": {
+            "id": "https://i40-aas-approval-app.cfapps.eu10.hana.ondemand.com/aas",
+            "idType": "IRI"
+        },
+        "endpoints": [
+            {
+                "url": "https://admin:admin123@i40-aas-approval-app.cfapps.eu10.hana.ondemand.com/register",
```

Comment:
What is our policy on publishing the credentials to our CF app? @Manu3756 @kasimrehman 
We agreed not to do this for now, didn't we?

Reply:
Indeed, the json should not be in the version control and was updated by mistake.
I changed it back to the version in canary